ailerons_tracker_backend/__init__.py

```diff
@@ -33,74 +34,91 @@ def create_app(test_config=None):
     except OSError:
         pass
 
-    @app.route('/upload', methods=['GET', 'POST'])
+    @app.post('/upload')
     def upload_file():
-        """ Parse a CSV file and insert data in DB
+        """ Parse a CSV file and insert data in DB """
 
-        Returns:
-            204: Successful operation
-            e: Error while attempting insert in DB """
+        try:
+            # A priori on aurait deux fichiers donc j'ai donné un nouveau nom à celui ci "
```

Comment:
I created a "prepare_csv" function so that we can retrieve multiple CSV files from the request.
For now it only gets the localization data file, named "locFile", creates a local copy and return the name and path of the file.